test/integration/scheduler/priorities_test.go

```diff
@@ -72,9 +71,9 @@ func TestNodeAffinity(t *testing.T) {
 	testCtx := initTestSchedulerForPriorityTest(t, nodeaffinity.Name)
 	defer testutils.CleanupTest(t, testCtx)
 	// Add a few nodes.
-	_, err := createNodes(testCtx.ClientSet, "testnode", st.MakeNode(), 4)
+	_, err := createAndWaitForNodesInCache(testCtx, "testnode", st.MakeNode(), 4)
 	if err != nil {
-		t.Fatalf("Cannot create nodes: %v", err)
+		t.Fatal(err)
```

Comment:
nit: revert. The extra context doesn't hurt. Same for other Fatal calls.

Reply:
I deliberately let `createAndWaitForNodesInCache()` return the error with context - failed to create nodes, or fail to obtain nodes in cache. So let's just use `err` in the caller side.